api/types/types.go

```diff
@@ -100,21 +100,6 @@ type Ping struct {
 	Experimental bool
 }
 
-// Version contains response of Engine API:
-// GET "/version"
-type Version struct {
-	Version       string
-	APIVersion    string `json:"ApiVersion"`
-	MinAPIVersion string `json:"MinAPIVersion,omitempty"`
```

Comment:
Ya, something is wrong here. It's working for other types. I need to investigate.